scratch2/draw.js

```diff
@@ -32,26 +32,29 @@ var SVG = (module.exports = {
   },
 
   el(name, props) {
-    var el = document.createElementNS("http://www.w3.org/2000/svg", name)
+    const el = document.createElementNS("http://www.w3.org/2000/svg", name)
     return SVG.setProps(el, props)
   },
 
   setProps(el, props) {
-    for (var key in props) {
-      var value = "" + props[key]
+    for (const key in props) {
+      const value = "" + props[key]
       if (directProps[key]) {
         el[key] = value
       } else if (/^xlink:/.test(key)) {
         el.setAttributeNS("http://www.w3.org/1999/xlink", key.slice(6), value)
-      } else if (props[key] !== null && props.hasOwnProperty(key)) {
+      } else if (
+        props[key] !== null &&
+        Object.prototype.hasOwnProperty.call(props, key)
```

Comment:
We should use `Object.prototype`. Please see also https://eslint.org/docs/rules/no-prototype-builtins